frontend/src/component/pages/shared/Header.scss

```diff
@@ -21,4 +21,11 @@ header {
     letter-spacing: 2px;
     line-height: 30px;
   }
+
+  .sign-out {
+    margin-left: auto;
+    button {
+      height: 100%;
+    }
```

Comment:
Let's open an issue and add `// TODO(issue#your_issue_number): remove dependency on internal implementation of the Button.` before those lines.